libcloud/compute/drivers/openstack.py

```diff
@@ -1672,7 +1691,7 @@ def ex_delete_network(self, network):
         resp = self.connection.request('%s/%s' % (self._networks_url_prefix,
                                                   network.id),
                                        method='DELETE')
-        return resp.status == httplib.ACCEPTED
+        return resp.status in (httplib.NO_CONTENT, httplib.ACCEPTED)
```

Comment:
even though the new `return resp.status in (httplib.NO_CONTENT, httplib.ACCEPTED)` is the correct behavior, wouldn't it make more sense to put this change in a separate pull request @micafer? This doesn't really have something to do with 'Implement new versions of OpenStack API calls' but this change will show up as such in the [CHANGES](https://github.com/apache/libcloud/blob/trunk/CHANGES.rst) file. 

Imagine that someone uses the ex_delete_network method in their software, happens to depend on the method returning False if NO_CONTENT is returned ([even though that is a bug](https://xkcd.com/1172/)), that causes a bug in their systems, and they then look in the CHANGES file for what might have caused the change in behavior. If this change is part of this PR it will be very hard to spot.

Reply:
I have to change this because the network api returns NO_CONTENT instead of ACCEPTED so without this change the function will return false with the new API: [delete network](https://developer.openstack.org/api-ref/network/v2/?expanded=delete-network-detail#delete-network)

Reply:
I understand, but you are changing this in the OpenStack_1_1_NodeDriver not only in the OpenStack_2_NodeDriver

Reply:
Ah I see, previously in the os-networks API this would never return NO_CONTENT, only ACCEPTED. https://developer.openstack.org/api-ref/compute/?expanded=delete-network-detail#delete-network In that case there is no problem.